docs/topics/native/native-definition-file.md

```diff
@@ -0,0 +1,268 @@
+[//]: # (title: Definition file)
+
+Kotlin/Native enables you to consume C and Objective-C libraries, allowing you to use their functionality in Kotlin.
+A special tool called cinterop takes a C or an Objective-C library and generates the corresponding Kotlin bindings,
+so that the library's methods can be used in your Kotlin code as usual.
+
+To generate these bindings, each library needs a definition file, usually with the same name as the library.
+This is a property file that describes exactly how the library should be consumed. See the full [list of available properties](#properties).
+
+Here's a general workflow when working with a project:
+
+1. Create a `.def` file describing what to include in the bindings.
+2. Use the generated bindings in your Kotlin code.
+3. Run the Kotlin/Native compiler to produce the final executable.
+
+## Create and configure a definition file
+
+Let's create a definition file and generate bindings for a C library:
+
+1. In your IDE, select the `src` folder and create a new directory with **File | New | Directory**.
+2. Name the new directory **nativeInterop/cinterop**.
```

Comment:
Don't we usually write directory names with inline code formatting?

Reply:
No, you're right